src/libraries/Microsoft.PowerFx.Interpreter/RecalcEngine.cs

```diff
@@ -60,22 +60,23 @@ private void AddInterpreterFunctions(PowerFxConfig powerFxConfig)
             powerFxConfig.AddFunction(BuiltinFunctionsCore.Value_UO);
             powerFxConfig.AddFunction(BuiltinFunctionsCore.Boolean);
             powerFxConfig.AddFunction(BuiltinFunctionsCore.Boolean_UO);
+
+            return powerFxConfig;
         }
 
-        /// <summary>
-        /// List all functions (both builtin and custom) registered with this evaluator. 
-        /// </summary>
-        public IEnumerable<string> GetAllFunctionNames()
```

Comment:
>GetAllFunctionNames [](http://example.com/codeflow?start=35&length=19)

subtle: this is just returning the functions *implemented* in the interpreter (which is good), not necessarily all the builtins.

Reply:
added test for this